packages/next/src/server-build.ts

```diff
@@ -981,6 +981,14 @@ export async function serverBuild({
     );
 
     for (const group of combinedGroups) {
+      // Skip group that only has an _error page
+      if (
+        group.pages &&
+        group.pages.length === 1 &&
+        group.pages[0] === '_error.js'
+      ) {
```

Comment:
This heuristic doesn't seem safe as a user could configure custom serverless function limits e.g. memory or duration for all pages except `_error` and then this would be missing the `_error` output unexpectedly.